test/fixtures/rules/css-modules/unused.js

```diff
@@ -0,0 +1,17 @@
+// unused.js
+import React from 'react';
+
+import badStyles from './test.css';
+// eslint-disable-next-line no-duplicate-imports
```

Comment:
Perhaps simply duplicate `test.css` to `test-copy.css`?

Reply:
i was trying to minimize the number of files but i will create a copy file then

Reply:
Alright, also just use `styles` and `stylesCopy` for the variable names instead of `badStyles` and `goodStyles`.